apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SeedStep.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Flex } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+// import { StorylyInstanceID } from "@ledgerhq/types-live";
+
+// import { StorylyWrapper } from "~/renderer/components/Storyly";
+import { StepText } from "./shared";
+
+export type SeedPathStatus =
+  | "choice_new_or_restore"
+  | "new_seed"
+  | "choice_restore_direct_or_recover"
+  | "restore_seed"
+  | "recover_seed";
```

Comment:
I have a personal preference for enums for these things as it avoids having to hardcode strings, though I'm sure there's a good reason to use union types over enums